app/components/ui/Modals/AdminPanel/components/FormatDescriptionForm.tsx

```diff
@@ -69,14 +72,14 @@ export function FormatDescriptionForm({ id, subTitle }: FormatDescriptionFormPro
     <h1 className="flex flex-row">
       <p className="hidden tablet:block">Description:&nbsp;</p>
       {isEditing ? (
-        <form onSubmit={handleSubmit(onSubmit)}>
+        <form className="w-full" onSubmit={handleSubmit(onSubmit)}>
           <div ref={inputRef}>
             <ProductInput
-              className={twMerge(`w-full text-center tablet:text-start`, isLoading && "animate-pulse")}
+              className={twMerge(`border w-full text-center tablet:text-start`, isLoading && "animate-pulse")}
               id="subTitle"
               register={register}
               errors={errors}
-              placeholder={subTitle}
+              placeholder={subTitle + " (use 'Shift + Enter' for a new line)"}
```

Comment:
I see that you also added in this commit
> " (use 'Shift + Enter' for a new line)"

Please think in advance what sub-tasks you should to do for this task (textarea in subTitle) and do it in separate commits - it helps me undestand all things you did